<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SBAS_time_series_analysis</class>
 <widget class="QWidget" name="SBAS_time_series_analysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SBAS时间序列分析</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../QtWidgetsApplication3.qrc">
    <normaloff>:/QtWidgetsApplication3/bin/icon/time_series.png</normaloff>:/QtWidgetsApplication3/bin/icon/time_series.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="2,2,2,2,1,1,1">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>输入输出设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>工程选择</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_project"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,7">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>数据节点选择</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_srcNode"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,7">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>保存数据节点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_dstNode"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>干涉相位生成设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>时间基线高阈值（天）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_temporal_thresh"/>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>时间基线低阈值（天）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_temporal_thresh_low"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>空间基线阈值（米）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_spatial_thresh"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>距离向多视倍数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_multilook_rg"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>方位向多视倍数</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_multilook_az"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Goldstein滤波强度</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_Goldstein_alpha"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>相位解缠设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>相位解缠方法</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_Delaunay_MCF">
          <property name="text">
           <string>Delaunay_MCF</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_SNAPHU">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>SNAPHU</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_MCF">
          <property name="text">
           <string>MCF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>相关系数阈值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_coherence_thresh"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>轨道精炼和重去平设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,3">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>重去平控制点相关系数阈值</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_reflattening_coh_thresh"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="7,3">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>重去平控制点形变速率阈值（m/year）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_reflattening_def_thresh"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>时间序列分析设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>模型相关系数阈值（temporal_coherence）</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBox_temporal_coherence_thresh"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
